### DISCUSSION TITLE
跟着Tutorial学qteasy

### DISCUSSION THREAD

**Discussion by ziarcher** [shepherdpp/qteasy#123] [Show and tell]
Jackie PENG您好，今天学会了jupyter notebook中运行代码，也遇到了一些问题，记录在附件里，空闲时帮看看，谢谢
[Tutorial 02记录.md](https://github.com/shepherdpp/qteasy/files/14476172/Tutorial.02.md)

**Comment by ziarcher** [comment]
非常感谢Jackie PENG，安装更新1.0.27后，上面问题已解决，程序按照教程就能跑起来，已下载一些金融数据到本地

**Comment by shepherdpp** [comment]
目前qteasy仍处于beta测试阶段，程序的很多功能还在积极地开发完善中，与依赖包和高版本python的各种兼容性问题也还在不断测试中，如您所见，Tutorial也还没有跟进覆盖到全部的功能。因此不免有bug或不足之处，感谢您指出来，我会通过不断地版本迭代开发更完善更稳定的版本。

目前即将发布1.1.0版本，这个新版本对实盘模拟交易下的命令行进行了不少改进和修正，期待更多的意见或建议以进一步改进！
